### ISSUE TITLE
[FEATURE] Dynamic focus

### ISSUE THREAD

**Issue by arBorghi** [michaellee/stille#23]
Just a simple feature.

Currently, the notes stay unfocused by default, and just when clicking on a block it gets highlighted. I suggest keeping the entire note on focus by default and just reducing text opacity when a text block is selected.
It will make it way more flexible and comfortable because it won't be necessary to turn it on/off that often.

**Comment by michaellee** [comment]
Hey @arBorghi thanks for creating this issue.

I'm not sure I fully understand what you're asking for.

So when Stille is "on" then the text will show in full capacity until you select a block will it focus out all the blocks around it?

**Comment by arBorghi** [comment]
> 

Yes, exactly. The idea is to keep everything with 100% opacity when no block is focused and dim everything else when something gets clicked on.
It's harder to focus on something when you can't even see it from the start. Everything gets the same visual hierarchy, but harder to spot.
